### ISSUE TITLE
ADD: Energy usage from statistics (

### ISSUE THREAD

**Issue by marcmanusch** [erikkastelec/hass-WEM-Portal#33]
It Would be great, if there are variables to see the used energy from the day. 

<img width="1546" alt="Bildschirmfoto 2022-08-25 um 12 07 01" src="https://user-images.githubusercontent.com/31078285/186637425-47ae0097-f24b-4da8-b78d-19f0ef822bbf.png">

**Comment by dm82m** [comment]
You can do that on your own: https://github.com/erikkastelec/hass-WEM-Portal/issues/19#issuecomment-1093707465

**Comment by marcmanusch** [comment]
No, there is no variable for it, that’s the reason for making the issue.
![ABD37824-F48B-4C08-A848-B8E54D189394](https://user-images.githubusercontent.com/31078285/187091718-e24c8355-71ba-47d2-b4e1-14a782f1869e.png)

**Comment by marcmanusch** [comment]
There is the value „sensor.leistungsabgabe“. This value can be used in Dashboards, but not in the energy dash. 

https://developers.home-assistant.io/docs/core/entity/sensor/#long-term-statistics

The sensor has wrong configuration, please fix it.

**Comment by dm82m** [comment]
Won’t help you. The data is kWh and the energy dashboard in HA used m3. Read my post linked above. All is described that how to solve your problem.

**Comment by vanto** [comment]
Since https://github.com/home-assistant/core/pull/58426 HA's energy dashboard also supports kWh as unit for gas.

**Comment by timg83** [comment]
First of all, thanks for this great HA integration!

Using the sensor "leistungsabgabe" I can see the current usage of the heatpump in HA. How can I see the daily usage, in a period of 24h, of the heatpump in HA? I have no gas so the unit should be kWh. I also like to know how I can add the heatpump usage as an individual device in the HA energy dashboard.
